utoipa-swagger-ui/build.rs

```diff
@@ -22,34 +22,40 @@ use zip::{result::ZipError, ZipArchive};
 const SWAGGER_UI_DOWNLOAD_URL_DEFAULT: &str =
     "https://github.com/swagger-api/swagger-ui/archive/refs/tags/v5.17.3.zip";
 
+const SWAGGER_UI_DOWNLOAD_URL: &str = "SWAGGER_UI_DOWNLOAD_URL";
+const SWAGGER_UI_OVERWRITE_FOLDER: &str = "SWAGGER_UI_OVERWRITE_FOLDER";
+
 fn main() {
     let target_dir = env::var("OUT_DIR").unwrap();
-    println!("OUT_DIR: {}", target_dir);
+    println!("OUT_DIR: {target_dir}");
 
     let url =
-        env::var("SWAGGER_UI_DOWNLOAD_URL").unwrap_or(SWAGGER_UI_DOWNLOAD_URL_DEFAULT.to_string());
+        env::var(SWAGGER_UI_DOWNLOAD_URL).unwrap_or(SWAGGER_UI_DOWNLOAD_URL_DEFAULT.to_string());
 
-    println!("SWAGGER_UI_DOWNLOAD_URL: {}", url);
+    println!("{SWAGGER_UI_DOWNLOAD_URL}: {url}");
     let zip_filename = url.split('/').last().unwrap().to_string();
     let zip_path = [&target_dir, &zip_filename].iter().collect::<PathBuf>();
 
-    if !zip_path.exists() {
```

Comment:
I assume the cache is still desired for HTTP(S) downloads.

Reply:
Yes: I understand that this is covered by `println!("cargo:rerun-if-env-changed={SWAGGER_UI_DOWNLOAD_URL}");
` [here](https://github.com/juhaku/utoipa/blob/403d716513612cb52bef0525640b9a96cd1bb18f/utoipa-swagger-ui/build.rs#L179), right ?

Reply:
Sort of, `rerun-if-env-changed` tells cargo about _more_ reasons to rebuild, while the old cache was trying to avoid redownloading when it _was_ rebuilt. (On the other hand, I'm not sure how useful that was anyway given that I'm not sure how Cargo decides on when to recreate out folders.)

Reply:
To be clear: I'm fine with the changed behaviour, just wanted to double check that it was intended.

Reply:
Yeah, I removed it just because it appears to be not constant on creating the folders thus even if there was the cache previously cargo might still create new set of folders when building thus invalidating the cache in a mean time as well. Perhaps just better of to let it download it when ever cargo should rebuild it. After all that is not going to be build unless one runs `cargo clean`.